.github/workflows/ecs.yml

```diff
@@ -0,0 +1,81 @@
+name: ecs
+
+on:
+  workflow_call:
+    inputs:
+      composer-command:
+        description: Composer command to run.
+        default: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
+        required: false
+        type: string
+      composer-version:
+        description: Composer version to use.
+        default:
+        required: false
+        type: string
+      coverage:
+        description: Generate code coverage report.
+        default: none
+        required: false
+        type: string
+      extensions:
+        description: List of extensions to PHP.
+        default:
+        required: false
+        type: string
+      ini-values:
+        description: Initial values for PHP configuration.
+        default: date.timezone='UTC'
+        required: false
+        type: string
+      os:
+        description: Os to test, separated by comma.
```

Comment:
```suggestion
        description: OS to run action, separated by comma.
```

Reply:
![image](https://github.com/yiisoft/actions/assets/42547589/52c35467-82b0-4935-afa0-4c994c682653)

This is how it is in all CI.

Reply:
ECS is not test

Reply:
I have no problem changing it, but then you have to adjust all the CI, so there is a standard.

Reply:
@vjik variant is more correct.